### DISCUSSION TITLE
Liquid Ajax Cart x Multiple language stores

### DISCUSSION THREAD

**Discussion by haroldao** [EvgeniyMukhamedjanov/liquid-ajax-cart#39] [Q&A]
Hi Evgeniy !

It seems like the AJAX is not working when a store is using multiple languages (ex : store.com/en) 👀.

When we add an item to cart, we are redirected to the cart page (ex : store.com/en/cart)

**Comment by EvgeniyMukhamedjanov** [comment]
Hey Harold, nice to see you here :-)
What version of the library do you use?
I've added Shopify Markets support in 1.10.2.

**Reply by haroldao** [reply]
v1.10.1

**Reply by haroldao** [reply]
Ok... so should I upgrade? (will be the issue be resolved?)

**Reply by EvgeniyMukhamedjanov** [reply]
Yeah, download the latest one -- it is 1.10.3 I guess.
There are no breaking changes -- only market support is added.
